library/kotlin/src/io/envoyproxy/envoymobile/StreamEmitter.kt

```diff
@@ -49,5 +48,5 @@ interface StreamEmitter : CancelableStream {
    * @throws EnvoyException when there is an exception ending the stream or sending trailers.
    */
   @Throws(EnvoyException::class)
-  fun close(trailers: Map<String, List<String>>?)
+  fun close(trailers: Map<String, List<String>>? = null)
```

Comment:
We discussed on @junr03's PR not to provide default values here so that it's clear to the caller how they're closing the stream. I think we should explicitly pass `trailers: null` from the emitter